docs/api.md

```diff
@@ -0,0 +1,31 @@
+# API development guidelines
+
+## API description
+
+We use [OpenAPI Specification 2.0](https://swagger.io/specification/v2/)
+to describe our API.
+
+This OAS 2.0 description is then used by:
+* [Dredd](https://github.com/apiaryio/dredd) to check our implementation
+* [Committee](https://github.com/interagent/committee) to validate input data
+* [SwaggerUI](https://swagger.io/tools/swagger-ui) to generate documentation
+
+## JSON generation
+
+We use [jBuilder](https://github.com/rails/jbuilder) to write our JSON views.
+
+We make jBuilder use [Oj](https://github.com/ohler55/oj) to parse and marshall JSON.
```

Comment:
Well, actually, jBuilder uses [MultiJSON](https://github.com/intridea/multi_json) which tries to load JSON engine in order of speed and starts by... `oj` :trophy: 

> MultiJSON tries to have intelligent defaulting. That is, if you have any of the supported engines already loaded, it will utilize them before attempting to load any. When loading, libraries are ordered by speed. First Oj, then Yajl, then the JSON gem, then JSON pure. If no other JSON library is available, MultiJSON falls back to OkJson, a simple, vendorable JSON parser.

_[source: [MultiJSON README file](https://github.com/intridea/multi_json)]_

Reply:
I don't think that it should change our decision, and I would still suggest
considering that this is an optimization.

Reply:
I just changed it to "We **may**", would that work for you?